tests/sidewinder/working/s2n-cbc/patches/cbc.patch

```diff
@@ -0,0 +1,86 @@
+diff --git a/tests/sidewinder/working/s2n-cbc/tls/s2n_cbc.c b/tests/sidewinder/working/s2n-cbc/tls/s2n_cbc.c
+index b37efb0..077d214 100644
+--- a/tests/sidewinder/working/s2n-cbc/tls/s2n_cbc.c
++++ b/tests/sidewinder/working/s2n-cbc/tls/s2n_cbc.c
+@@ -25,6 +25,28 @@
+ 
+ #include "tls/s2n_record.h"
+ 
++#include <smack.h>
++#include <smack-contracts.h>
++#include "ct-verif.h"
++#include "sidewinder.h"
++
++/* Because of a bug in SMACK, when we try to specify the invarients for this loop in the main function, 
++ * SMACK crashes. Moving the code (unmodified) to this function sidesteps the bug. Michael Emmi has promissed
++ * a fix soon
```

Comment:
Is there a Github issue that can be linked here?

Reply:
I have made them aware, and will send them a minimal reproducing case. 
 https://github.com/awslabs/s2n/issues/686 tracks this, and integrating their fix